tests/core/pyspec/eth2spec/test/merge/fork_choice/test_terminal_validity.py

```diff
@@ -0,0 +1,176 @@
+from eth2spec.utils.ssz.ssz_typing import uint256
+from eth2spec.test.helpers.block import (
+    prepare_empty_pow_block
+)
+from eth2spec.test.context import spec_state_test, with_merge_and_later
+
+
+def create_transition_store(spec):
+    anchor_block = prepare_empty_pow_block(spec)
+    transition_store = spec.get_transition_store(anchor_block)
```

Comment:
`transition_store` was removed from spec. Will need to clean this up a bit after a rebase

Reply:
A lot of this is changed, so I'll review after rebase